src/managers/continuous/index.js

```diff
@@ -390,7 +390,7 @@ class ContinuousViewManager extends DefaultViewManager {
 		let prepend = () => {
 			let first = this.views.first();
 
-			if (first?.section && first.section.idref.includes(EMPTY_PAGE_STRING) && !this.ignoreAutoProgress) {
+			if (first && first.section && first.section.idref.includes(EMPTY_PAGE_STRING) && !this.ignoreAutoProgress) {
```

Comment:
This was just a linting complaint unrelated. But I had to fix it in order to do yarn linking